Content.Server/Xenoarchaeology/XenoArtifacts/Triggers/Systems/ArtifactMagnetTriggerSystem.cs

```diff
@@ -22,28 +22,28 @@ public override void Update(float frameTime)
     {
         base.Update(frameTime);
 
-        var artifactQuery = EntityQuery<ArtifactMagnetTriggerComponent, TransformComponent>().ToHashSet();
-        if (!artifactQuery.Any())
+        if (!EntityQuery<ArtifactMagnetTriggerComponent>().Any())
             return;
```

Comment:
Previously I would've said Count<T> though I think it's slower with Arch but we should get Any<T> on engine.